### DISCUSSION TITLE
Brighter Version for New Development

### DISCUSSION THREAD

**Discussion by SteveBush** [BrighterCommand/Brighter#3253] [Q&A]
With the implementation of a host of new features, the 9.x and 10.x branches have really diverged.  I'm starting a new messaging project, and I'm trying to determine if I should wait until 10.x is released or use 9.X and plan on reimplementing on 10.x later.  

What is the timing of the 10.x release out of preview? 10.0.0.preview-3 was released 5 months ago.

I'm willing to contribute to helping move forward with the 10.0.0 release, as it will save me time in the future.

Thanks.

**Comment by PleasantD** [comment]
I'm also interested in the timeline for v10.
I'm currently looking into implementing some new features using message queues and the outbox pattern.
With MassTransit imminently moving to a commercial license, other free options are being considered.

This project looks to have a lot of the features that we would need, but I would be sad to have to heavily refactor new work in few weeks/months.

**Comment by iancooper** [comment]
We have just released RC1. We aim to release RC2 by the end of the month (JULY 2025). RC2 will be feature complete, so the only thing preventing it from being final will be significant bugs. We hope to declare V10 complete by early August.
